### PR TITLE
refactor: fix and rewrite form field styles according to specification

### PR DESCRIPTION
**Pull request by jeripeierSBB**

### CODE REVIEW

**Review by federicoisepponfincons — COMMENTED** [review]
everything looks fine, there are just two points to discuss

**Review by federicoisepponfincons — APPROVED** [review]

File: src/components/sbb-form-field/sbb-form-field.scss [hunk 1]
```diff
@@ -6,36 +6,43 @@
 :host {
   display: inline-block;
```
[the remark this answers is not in the document]
Comment (jeripeierSBB) [reply src/components/sbb-form-field/sbb-form-field.scss:7]: As I said, I think its better to go with inline-block. Final decision @kyubisation

[the remark this answers is not in the document]
Comment (kyubisation) [reply src/components/sbb-form-field/sbb-form-field.scss:7]: I agree with inline-block. It aligns with the native behavior of form elements like input and select.
With this it is still easy for consumers to switch to a vertical alignment (e.g. with flexbox).

File: src/components/sbb-form-field/sbb-form-field.scss [hunk 2]
```diff
@@ -63,59 +72,65 @@
...
 }
 
+::slotted(*[slot='label']),
+slot[name='label'] > span {
+  @include ellipsis;
+}
+
 // Input
 /* stylelint-disable plugin/stylelint-bem-namics */
 .form-field__input ::slotted(:is(input, select, .sbb-field-input)) {
```
[the remark this answers is not in the document]
Comment (jeripeierSBB) [reply src/components/sbb-form-field/sbb-form-field.scss:132]: discussed with @kyubisation. We leave !important for the moment.

**Pull request merged**